app/views/donations/new.html.erb

```diff
@@ -52,7 +55,12 @@
   document.getElementById("donation_button").addEventListener("click", myFunction);
 
   function myFunction() {
-    document.getElementById("modal-amount").innerText = 'Valor: ' + document.getElementById("donation_amount").value ;
-    document.getElementById("modal-type").innerText = 'Tipo: ' + document.getElementById("donation_payment_type").value ;
+    const selected = document.querySelector(".tag-selector:checked")
+    if (selected != null && selected.value != "") {
+      document.getElementById("modal-amount").innerText = 'Valor: R$ ' + document.querySelector(".tag-selector:checked").value;
+    } else {
```

Comment:
Quando o usuário seleciona ou escreve um número aqui funciona muito bem, mas e se nada for selecionado nem escrito? Temos que retornar um erro pro usuário.

Eu ainda sou da opinião que poderíamos deixar o simple form lidar com os erros e ao invés de fazer o modal a gente passa pra próxima página de checkout... Mas é só minha opinião mesmo, discorde numa boa!